apps/common-app/src/examples/LayoutAnimations/LayoutTransitionExample.tsx

```diff
@@ -0,0 +1,295 @@
+import React, { useState, useRef, useEffect } from 'react';
+import {
+  View,
+  Text,
+  Button,
+  SafeAreaView,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+import Animated, {
+  LinearTransition,
+  SequencedTransition,
+  FadingTransition,
+  FadeOut,
+  JumpingTransition,
+  CurvedTransition,
+  EntryExitTransition,
+  FlipInEasyX,
+  FlipOutYRight,
+} from 'react-native-reanimated';
+import { createStackNavigator } from '@react-navigation/stack';
+import { NavigationContainer } from '@react-navigation/native';
+import { type StackScreenProps } from '@react-navigation/stack';
+
+interface Item {
+  id: number;
+  content: string;
+  color: string;
+}
+
+const INITIAL_LIST = [
+  { id: 1, content: '🍌', color: '#b58df1' },
+  { id: 2, content: '🍎', color: '#ffe780' },
+  { id: 3, content: '🥛', color: '#fa7f7c' },
+  { id: 4, content: '🍙', color: '#82cab2' },
+  { id: 5, content: '🍇', color: '#fa7f7c' },
+  { id: 6, content: '🍕', color: '#b58df1' },
+  { id: 7, content: '🍔', color: '#ffe780' },
+  { id: 8, content: '🍟', color: '#b58df1' },
+  { id: 9, content: '🍩', color: '#82cab2' },
+];
+
+type RootStackParamList = {
+  Home: undefined;
+  linear: { title: string; transition: any };
+  fading: { title: string; transition: any };
+  sequenced: { title: string; transition: any };
+  jumping: { title: string; transition: any };
+  curved: { title: string; transition: any };
+  entryexit: { title: string; transition: any };
```

Comment:
I'm aware that we don't have any public type for transitions, but I'm considering whether replacing `any` with `unknown` would be a better choice. What are your thoughts on this?

Reply:
As we discussed on dms, it's not possible to use `unknown` - `any` stays 🆗